model/host/host_test.go

```diff
@@ -4622,6 +4622,46 @@ func TestCountSpawnhostsWithNoExpirationByUser(t *testing.T) {
 	assert.Equal(t, 0, count)
 }
 
+func TestCountIntentHosts(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Instead of `ctx, cancel` could you use `t.Context()`? It's in the new Go version and it cuts down on boiler plate for tests.